### PR TITLE
Add total source count of all connectors

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
Addresses Unified connectors

## Changes

-Add total source count of all connectors

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [x] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/unified_connector/schema.py [hunk 1]
```diff
@@ -31,6 +31,13 @@
 )
 
 
+def get_total_source(info):
+    count = ConnectorSourceLead.objects.filter(source__unified_connector__project=info.context.active_project).count()
+    if PP.check_permission(info, PP.Permission.VIEW_UNIFIED_CONNECTOR):
+        return count
+    return
```
Comment (thenav56) [inline apps/unified_connector/schema.py]: We don't have to do the count operation if the user doesn't have permission.

```suggestion
def get_total_source(info):
    qs = ConnectorSourceLead.objects.filter(source__unified_connector__project=info.context.active_project)
    if PP.check_permission(info, PP.Permission.VIEW_UNIFIED_CONNECTOR):
        return qs.count()
    return
```

File: apps/unified_connector/schema.py [hunk 2]
```diff
@@ -215,6 +222,11 @@ class UnifiedConnectorQueryType(graphene.ObjectType):
             page_size_query_param='pageSize',
         )
     )
+    total_source_count = graphene.Field(graphene.Int)
+
+    @staticmethod
+    def resolve_total_source_count(root, info, **kwargs):
+        return get_total_source(info)
```
Comment (thenav56) [inline apps/unified_connector/schema.py]: This should be fine here.
```suggestion
           qs = ConnectorSourceLead.objects.filter(source__unified_connector__project=info.context.active_project)
           if PP.check_permission(info, PP.Permission.VIEW_UNIFIED_CONNECTOR):
               return qs.count()
           return
```

**Review by thenav56 — CHANGES_REQUESTED** [review]
Looks good just view a few comments.

**Review by thenav56 — APPROVED** [review]

**Comment by AdityaKhatri** [comment]
Closed because: https://github.com/the-deep/server/pull/1169

**Pull request closed without merging**